### PR TITLE
Drop Token:Rss

### PR DESCRIPTION
**Pull request by hennevogel**
Depends on #15238 to be merged and deployed...

### CODE REVIEW

**Comment by hennevogel** [comment]
#15238 is merged but not deployed yet

**Comment by hennevogel** [comment]
#15238 is deployed for a while now...

File: src/api/db/data/20231208145155_drop_token_rss.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DropTokenRss < ActiveRecord::Migration[7.0]
+  def up
+    Token.where(type: 'Token::Rss').delete_all
```
[the remark this answers is not in the document]
Comment (hennevogel) [reply src/api/db/data/20231208145155_drop_token_rss.rb:5]: There are no `EventSubscriptions` for Token::Rss, they work the other way around...

[the remark this answers is not in the document]
Comment (dmarcoux) [reply src/api/db/data/20231208145155_drop_token_rss.rb:5]: Ah okay, I didn't know, then it's fine.

**Review by dmarcoux — APPROVED** [review]

File: src/api/db/data/20231208145155_drop_token_rss.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (hennevogel) [reply src/api/db/data/20231208145155_drop_token_rss.rb:5]:
```
EventSubscription.where(channel: :rss).where.not(token_id: nil).count
=> 0
```

**Pull request merged**